v3/docs/00-style-and-contributor-guidelines/b-main-style-guide/index.mdx

```diff
@@ -429,10 +429,8 @@ your life and the maintainers lives easier.
 
 ### New pages and moving files
 
-- The `i18n` folder is used for translation **but also for the rendering of the navigation**
-  **elements sections**. Your `index.mdx` file's `title` is not the source for this.
 - If you are adding or renaming a page, you **must** add it correctly in `src/components/DevNavMenu.tsx` and
```

Comment:
@sacha-l there will be one step less needed now when adding new material as it does not source translation strings anymore. All is now in `src/components/DevNavMenu.tsx`

I've updated this guide to reflect the change.